src/domain/users/index.types.d.ts

```diff
@@ -52,6 +52,7 @@ type User = {
   deviceTokens: DeviceToken[]
   phoneMetadata: PhoneMetadata | undefined
   phone?: PhoneNumber | undefined
+  deletedPhone?: PhoneNumber[]
```

Comment:
shouldn't this be plural? `deletedPhones`?

Reply:
good point I'll update

Reply:
good point I'll update